.github/workflows/internal_tools.yml

```diff
@@ -3,22 +3,21 @@ name: Install internal tools Python packages & run test suite
 on:
   pull_request:
     branches: [main]
-  push:
-    branches: [main]
-  merge_group:
-    types: [checks_requested]
+    paths:
+      - internal/infra-bionemo/**
+
 jobs:
   infra-bionemo:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          submodules: 'recursive'
+          submodules: "recursive"
       - uses: actions/setup-python@v5
         with:
-          python-version: '3.10'
-          cache: 'pip'
+          python-version: "3.10"
```

Comment:
This doesn't match our current image of 3.12

Reply:
These run outside the image; this was just my autoformatter running